### PR TITLE
Fixed PHP sample app.

### PR DESCRIPTION
**Pull request by ktamashun**
* GRPC extension is now enabled.
* "credentials" key is now added to the $ops parameter.
* Updated dependencies.

### CODE REVIEW

**Comment by ktamashun** [comment]
I signed it!

File: 1.0/php/sample_app/composer.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "description": "gRPC example for PHP",
   "require": {
     "php": ">=5.5.0",
-    "grpc/grpc": "v1.0.0"
+    "grpc/grpc": "v1.2.0",
+    "stanley-cheung/protobuf-php": "v0.6"
```
Comment (stanley-cheung) [inline 1.0/php/sample_app/composer.json]: This `stanley-cheung/protobuf-php` dependency is no longer needed after grpc version 1.1.0+

**Review by stanley-cheung — COMMENTED** [review]
- In `1.0/php/Dockerfile`, the block building `Protobuf-PHP` is no longer needed
 - But on the other hand, we need 3.2.0+ version of the `protoc` compiler in the block above.

File: 1.0/php/sample_app/composer.json (hunk 1 above)
Comment (ktamashun) [reply 1.0/php/sample_app/composer.json]: You're right, I removed it. Thx!

**Review by stanley-cheung — COMMENTED** [review]
Looking good. Can you please squash the commits into one please?

**Comment by ktamashun** [comment]
Squashed the commits. Thx!

**Review by stanley-cheung — APPROVED** [review]
LGTM, Thanks for the contribution!

**Pull request merged**